mail_tracking/static/src/components/failed_message/failed_message.esm.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module **/
+
+import {Message} from "@mail/components/message/message";
+import {registerMessagingComponent} from "@mail/utils/messaging_component";
+
+export class FailedMessage extends Message {}
+
+Object.assign(FailedMessage, {
```

Comment:
Shouldn't you use the native method `patch` here?

CC @rolandojduartem

Reply:
I think no, it seems the approach is creating a new component using the functionalities of Message. Remember patch is to change the behavior of the inherited component, if you use patch, youl will change the props and template of Message. Now I think Object.assign is not needed, just:

```suggestion
FailedMessage.template ="mail_tracking.FailedMessage";
```

As Odoo does [here](https://github.com/odoo/odoo/blob/6bb16bf8818042b05ff8d40f852fa8e7d9eba92f/addons/account/static/src/components/account_move_form/account_move_form.js#L33).